style/ember/README.md

```diff
@@ -2,3 +2,11 @@ Ember
 =====
 
 * Don't put a space between the opening handlebars braces and the variable.
+
+Ember-Data
+----------
+
+* Visually separate relationships (`DS.{hasMany,belongsTo}`) from attributes
+  (`DS.attr`) with newlines. [Example][relationships]
```

Comment:
I'm not sure if we should show `DS.{hasMany,belongsTo}` and `DS.attr` here, most people will understand what we mean and if they don't we have the sample to guide them.